inductiva/resources/machines_base.py

```diff
@@ -180,7 +177,7 @@ def terminate(self, **kwargs):
                     id=self.id,
                     name=self.name,
                     machine_type=self.machine_type,
-                    disk_size_gb=self._true_disk_size_gb,
+                    disk_size_gb=self.data_disk_gb,
```

Comment:
Since now we are not sending `_true_disk_size_gb` but `data_disk_gb` and the backend handles the rest would it make sense to also rename the arguments of the functions making the requests accordingly. That is:

```python
inductiva.client.models.GCPVMGroup(
                    id=self.id,
                    name=self.name,
                    machine_type=self.machine_type,
                    disk_size_gb=self._true_disk_size_gb,
                    # Change to data
                    data_size_gb=self.data_disk_gb,)
```

Reply:
_true_disk_size_gb no longed exists. All the math is handled on the back-end.